packages/trader/src/AppV2/Utils/trade-types-utils.tsx

```diff
@@ -0,0 +1,235 @@
+    ),
+};
+
+export const getTermDefinition = ({ contract_type, term }: { contract_type: string; term: string }) => {
+    const result = DEFINITION[term];
+    if (typeof result === 'function') return result(contract_type);
+    return result ?? '';
+};
+
+export const parseContractDescription = (
+    content: {
+        type: string;
+        text: JSX.Element | string;
+    }[]
+) =>
+    content.map(({ type, text }, index) => {
+        if (type === 'heading' && typeof text !== 'string')
+            return (
+                <Text key={text.props.i18n_default_text} bold size='md' className='description__heading'>
+                    {text}
+                </Text>
+            );
+        if ((type === 'paragraph' || type === 'general') && typeof text !== 'string')
+            return (
+                <Text
+                    as='p'
+                    key={text.props.i18n_default_text}
+                    size='sm'
+                    className={`description__${type === 'paragraph' ? 'paragraph' : 'general'}`}
+                    color='quill-typography__color--prominent'
+                >
+                    {text}
+                </Text>
+            );
+        if (type === 'video' && typeof text === 'string')
+            return <VideoFragment contract_type={text} key={text + index} />;
+    });
+
+/* The video upload feature is not available yet. The following video ids are taken from CF Stream account.
+        If considered necessary later, the current approach can be replaced with HTTP-request to fetch videos by their file names. */
+export const DESCRIPTION_VIDEO_ID: TDtraderVideoUrl = {
+    [CONTRACT_LIST.ACCUMULATORS]: {
+        light: 'c1d64d5ffaf449e3326d387a70621a4c',
+        dark: '9657e4adcfd9274d41a042b7b6c42d60',
+    },
+    [CONTRACT_LIST['EVEN/ODD']]: {
+        light: 'a3930b0a535c4f23daea5cf98a718941',
+        dark: 'e311e25dd68761d6ab73e8158ec83ea4',
+    },
+    [CONTRACT_LIST['HIGHER/LOWER']]: {
+        light: 'f528c662c377601908a741ae2aedaec6',
+        dark: '4c05445e1bd28f3f9f95b0ab566c4f9b',
+    },
+    [CONTRACT_LIST['MATCHES/DIFFERS']]: {
+        light: '8693a02397d284ffcf0608e4fda702d9',
+
+        dark: '2e56ad7ea67bf3f9a869de6336df1796',
+    },
+    [CONTRACT_LIST.MULTIPLIERS]: {
+        light: 'bca032791da88023e81ebf7341226f83',
+        dark: '3d10e25e2e0eaaf96d1874aae257029f',
+    },
+    [CONTRACT_LIST['OVER/UNDER']]: {
+        light: '9e2be907cba9b38352890e52cfd8cbaf',
+
+        dark: '4b31fc188c2f365faa310f7e34715af7',
+    },
+    [CONTRACT_LIST['RISE/FALL']]: {
+        light: '7719c7e5436f58e59ab47510445108ba',
+        dark: 'b72ef7ec914afe3dbb2a1601cad3a76f',
+    },
+    [CONTRACT_LIST['TOUCH/NO TOUCH']]: {
+        light: 'c781a6844f37a3308fe8774b4450dfc9',
+        dark: 'c91f2b9859dc95ce8eecf7df327aaf00',
+    },
+    [CONTRACT_LIST.VANILLAS]: {
+        light: '9b8b6ae67d3a720e4aee2665889d17fb',
+        dark: '8f04e9d768b8e08a1d87830c95c6b9c8',
+    },
+};
```

Comment:
I would suggest re-using an existing `DESCRIPTION_VIDEO_ID` from `video_config.ts` which is located in the same package and should be importable instead of copying all these IDs. :)
I know keys are different there but we can substitute keys from that config with the ones we need:
```

// function:
const replaceKeys = (config: TDtraderVideoUrl) => {
    return Object.fromEntries(Object.entries(config).map(([key, value]) => [CONTRACT_LIST[key.toUpperCase()], value]));
}

// usage of the existing DESCRIPTION_VIDEO_ID in getDescriptionVideoId:
export const getDescriptionVideoId = (contract_type = '', is_dark_theme = false) =>
    replaceKeys(DESCRIPTION_VIDEO_ID)[contract_type]?.[is_dark_theme ? 'dark' : 'light'];

```

for this to work, it's better to update keys for `CONTRACT_LIST` to match the keys from the existing config:

```
export const CONTRACT_LIST = {
    ACCUMULATOR: 'Accumulators',
    EVEN_ODD: 'Even/Odd',
    HIGH_LOW: 'Higher/Lower',
    MATCH_DIFF: 'Matches/Differs',
    MULTIPLIER: 'Multipliers',
    OVER_UNDER: 'Over/Under',
    RISE_FALL: 'Rise/Fall',
    TOUCH: 'Touch/No Touch',
    TURBOS: 'Turbos',
    VANILLA: 'Vanillas',
};
```

this way, we can remove the above copy of DESCRIPTION_VIDEO_ID. :)

Reply:
Replaced fields with the corresponding from old `DESCRIPTION_VIDEO_ID`